geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIndex.scala

```diff
@@ -41,7 +41,7 @@ case object AttributeIndex extends AccumuloAttributeIndex {
 }
 
 // secondary z-index
-trait AccumuloAttributeIndex extends AccumuloFeatureIndex with AccumuloIndexAdapter
+trait AccumuloAttributeIndex extends AccumuloFeatureIndex with AccumuloIndexAdapter[Unit]
```

Comment:
Alright, I'll bite.  What's the point of passing around [Unit] key types?  

I wonder if there's a sane type-level equivalent of option[T].

Reply:
yeah, the functionality isn't used in the class, so I just used Unit as the type binding...